FreeTubeApp/FreeTube
# Replace labeler workflow with GH labeler

# Replace labeler workflow with GH labeler




## Pull Request Type


## Related issue



All issues being opened within the past 2 months were being affected by this.

## Description

Our labeler didn't apply labels to issues. The maintainer of that workflow abandoned the project leaving us with a broken label workflow. I switched out the broken labeler with an issue labeler workflow from GH.

We have 2 workflows that by default look for labeler.yml in .github directory

- This workflow
- pr label workflow

To prevent both workflows looking at the same file i changed the name of the labeler.yml workflow in .github directory to pr-labeler.yml and pointed to that file in the workflow and did the same with this workflow.


## Testing 




1. Go to https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/FreeTube/issues/new/choose
2. Use the keywords while creating the issue
3. See labels being applied when issue is created

If u dont want to test yourself see my test issues: 

- https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/FreeTube/issues/38 
- https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/FreeTube/issues/37
